apps/docs/content/components/Section/index.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: Section
+status: Experimental
+source: https://github.com/primer/brand/tree/main/packages/react/src/Section/Section.tsx
+storybook: '/brand/storybook/?path=/story/components-section--default'
+description: Use the section component to create a semantic section in the page and group related content together.
+a11yReviewed: false
+---
+
+import {PropTableValues} from '../../../src/components'
+import {
+  SectionBackgroundColors,
+  SectionPaddingVariants,
+} from '@primer/react-brand'
+
+```js
+import {Section} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+The default section wraps its content in a semantic section element and provides default paddings.
+
+```jsx live
+<Section>
+  <SectionIntro align="center">
+    <SectionIntro.Heading size="3">Default section</SectionIntro.Heading>
+    <SectionIntro.Description>
+      This content is wrapped in a Section component.
+    </SectionIntro.Description>
+  </SectionIntro>
+</Section>
+```
+
+### With indepenedent block paddings
+
+```jsx live
+<Section paddingBlockStart="none" paddingBlockEnd="spacious">
+  <SectionIntro align="center">
+    <SectionIntro.Heading size="3">Custom paddings</SectionIntro.Heading>
+    <SectionIntro.Description>
+      This section has a custom padding at the top and bottom.
+    </SectionIntro.Description>
+  </SectionIntro>
+</Section>
+```
+
+### With predefined background color
+
+```jsx live
+<Section backgroundColor="inset">
```

Comment:
Can we change this to subtle as that's a more frequently used than inset, which i'm hopeful we can drop after v8 lands

```suggestion
<Section backgroundColor="subtle">
```